src/config.h.in

```diff
@@ -424,6 +423,12 @@
 /* Define if there is a working gettext(). */
 #undef HAVE_GETTEXT
 
+/* Define if there is a working bind_textdomain_codeset(). */
+#undef HAVE_BIND_TEXTDOMAIN_CODESET
```

Comment:
Is there a missing configure check for this `HAVE_BIND_TEXTDOMAIN_CODESET`?

Reply:
No, it was already there, but I moved it closer to the ``HAVE_GETTEXT`` and ``HAVE_DGETTEXT`` definitions (they are used together)

Reply:
I was talking about `configure.ac`. I only see a test for `dgettext()`, but not for `bind_textdomain_codeset()`?

Reply:
It's in configure.ac#L4499. I didn't change that line...

Reply:
ah thanks!